onionperf/measurement.py

```diff
@@ -389,6 +433,8 @@ def __start_tor_server(self, control_port, socks_port, hs_port_mapping):
     def __start_tor(self, name, control_port, socks_port, hs_port_mapping=None):
         logging.info("Starting Tor {0} process with ControlPort={1}, SocksPort={2}...".format(name, control_port, socks_port))
         tor_datadir = "{0}/tor-{1}".format(self.datadir_path, name)
+        key_path_v2 = os.path.expanduser("~/os_key_v2") 
```

Comment:
I don't like these paths. There is already a data directory created. We shouldn't just drop random files into home directories, especially private keys.

Reply:
One problem though is that the data directory currently does not contain anything that might be considered private, and we do want to make the logs available via the web server, so we might need to make an onionperf private directory to complement the data directory.

Reply:
Now onionperf makes a private directory by default (onionperf-private) to store the private keys for the onion services. This directory is on the same level as onionperf-data. Just like the current onionperf-data directory, a path to it can be passed as an argument on the command line.